README.md

```diff
@@ -107,7 +107,8 @@ Keep-Alive: timeout=5
 | -------------   |----------------------     |----------     |:--------:|---------------|
 | unleashUrl      |`UNLEASH_URL`              | n/a           | yes      | API Url to the Unleash instance to connect to |
 | unleashApiToken | `UNLEASH_API_TOKEN`       | n/a           | yes      | API token (client) needed to connect to Unleash API. |
-| proxySecrets    | `UNLEASH_PROXY_SECRETS`   | n/a           | yes      | List of proxy secrets the proxy accept. Proxy SDKs needs to set the Proxy secret as the `Authorization` heder when querying the proxy | 
+| clientKeys      | `UNLEASH_CLIENT_KEYS`     | n/a           | yes      | List of proxy secrets the proxy accept. Proxy SDKs needs to set the Proxy secret as the `Authorization` header (or another specified under `clientKeysHeaderName` option) when querying the proxy | 
```

Comment:
@thomasheartman renaming the `proxySecrets` option to `clientKeys`, as this causes a lot of confusion sense the SDKs call this a "clientKey". We are still maintaining backward compatibility, but we should update the docs after the proxy is released to drive adoption of the new field.

Reply:
Got it. It's also called the `clientSecret` in some SDKs. Should we update those too to keep them all in sync?

Reply:
Yes, I was about to check the native SDKs. We should avoid calling it a "secret" as long at it is not expected to be kept as a secret, but rather quite public to all users. I will update this line in the README.md as well to not use the word "secret".